src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int SCALING_NUMBER = 2;
+    private Object[] table;
+    private int size;
+    private int threshold;
```

Comment:
current threshold can be calculated based on size and array length, not need to have separate variables for them
```suggestion
```